beta/do-app-platform-op-cli/README.md

```diff
@@ -0,0 +1,145 @@
+# [Beta] Deploy 1Password SCIM bridge on DigitalOcean App Platform with 1Password CLI
+
+This deployment example describes how to deploy 1Password SCIM bridge as an app on DigitalOcean's [App Platform](https://docs.digitalocean.com/products/app-platform/) service using [1Password CLI](https://developer.1password.com/docs/cli), the DigitalOcean command line interface ([`doctl`](https://docs.digitalocean.com/reference/doctl/)), and the DigitalOcean [1Password Shell Plugin](https://developer.1password.com/docs/cli/shell-plugins/).
+
+The app consists of two [resources](https://docs.digitalocean.com/glossary/resource/): a [service](https://docs.digitalocean.com/glossary/service/) for the SCIM bridge container and an [internal service](https://docs.digitalocean.com/glossary/service/#internal-services) for Redis.
+
+## In this folder
+
+- [`README.md`](./README.md): the document that you are reading. 👋😃
+- [`op-scim-bridge.yaml`](./op-scim-bridge.yaml): an App Platform [app spec](https://docs.digitalocean.com/glossary/app-spec/) for 1Password SCIM bridge [templated with secret references](https://developer.1password.com/docs/cli/secrets-template-syntax) to load secrets from your 1Password account.
+
+## Overview
+
+Deploying 1Password SCIM bridge on App Platform comes with a few benefits:
+
+- For standard deployments, App Platform will host your SCIM bridge for a predictable cost of $10 USD/month (at the time of last review).
+- You do not need to manage a DNS record. DigitalOcean automatically provides a unique URL for your SCIM bridge.
+- You will deploy 1Password SCIM bridge directly to DigitalOcean from your local terminal. There is no requirement to clone this repository for this deployment.
+
+## Prerequisites
+
+- A 1Password account with an active 1Password Business subscription or trial
+  > **Note**
+  >
+  > Try 1Password Business free for 14 days: <https://start.1password.com/sign-up/business>
+- A DigitalOcean account with available quota for two droplets
+  > **Note**
+  >
+  > If you don't have a DigitalOcean account, you can sign up for a free trial with starting credit: <https://try.digitalocean.com/freetrialoffer/>
+- A Mac or Linux terminal with Bash, Zsh, or Fish
+<!--
+TODO: Investigate Windows Terminal, Powershell, WSL
```

Comment:
We don't support Windows yet for Shell Plugins

Reply:
Yeah, I know. This is a note for me to potentially document a workaround for people working from Windows environments (e.g. auth to `doctl` without using the shell plugin).

Reply:
Ah, gotcha